### PR TITLE
thingy91: Adding missing documentation

### PR DESCRIPTION
**Pull request by crfosse**
This commit adds documentation that should have been included with
[PR-5785](https://github.com/nrfconnect/sdk-nrf/pull/5785).

Signed-off-by: Carl Richard Steen Fosse <carl.fosse@nordicsemi.no>

### CODE REVIEW

File: samples/bluetooth/hci_lpuart/README.rst [hunk 1]
```diff
@@ -42,6 +42,12 @@ Testing
 
 The methodology to use to test this sample depends on the host application.
 
+Tips
+****
+
+This sample is also supported on the Thingy:91. However, it must be programmed using a debugger and a 10-pin SWD cable.
+Firmware updates over serial using MCUBoot is not supported for either of the MCUs on the Thingy:91 in this configuration.
+
```
Comment (umapraseeda) [inline samples/bluetooth/hci_lpuart/README.rst:50]: I think we can make this as part of the overview section above in both the samples.
```suggestion
This sample is also supported on the Thingy:91.
However, it must be programmed using a debugger and a 10-pin SWD cable.
Firmware updates over serial using MCUboot is not supported for either of the MCUs on the Thingy:91 in this configuration.

```

File: samples/bluetooth/hci_lpuart/README.rst [hunk 2]
```diff
@@ -21,6 +21,12 @@ Overview
 
 The sample implements the Bluetooth® HCI controller using the :ref:`uart_nrf_sw_lpuart` for UART communication.
 
+This sample is also supported on the Thingy:91.
+
+However, it must be programmed using a debugger and a 10-pin SWD cable.
+
+Firmware updates over serial using MCUboot are not supported for either of the MCUs on the Thingy:91 in this configuration.
```
Comment (divipillai) [inline samples/bluetooth/hci_lpuart/README.rst:28]:
```suggestion
This sample is also supported on the Thingy:91.
However, it must be programmed using a debugger and a 10-pin SWD cable.
Firmware updates over serial using MCUboot are not supported for either of the MCUs on the Thingy:91 in this configuration.
```
Please remove the extra line in between the sentences.

File: samples/nrf9160/lte_ble_gateway/README.rst [hunk 3]
```diff
@@ -35,6 +35,12 @@ When the connection is established, it starts collecting data from two sensors:
 The sample aggregates the data from both sensors in memory.
 You can then trigger an alarm that sends the aggregated data over LTE to `nRF Cloud`_ by flipping the Thingy:52, which causes a change in the flip state to ``UPSIDE_DOWN``.
 
+This sample is also supported on the Thingy:91.
+
+However, it must be programmed using a debugger and a 10-pin SWD cable.
+
+Serial communication and firmware updates over serial using MCUboot are not supported in this configuration.
```
Comment (divipillai) [inline samples/nrf9160/lte_ble_gateway/README.rst:42]:
```suggestion
This sample is also supported on the Thingy:91.
However, it must be programmed using a debugger and a 10-pin SWD cable.
Serial communication and firmware updates over serial using MCUboot are not supported in this configuration.
```
Please remove the extra line in between the sentences.

**Review by divipillai — APPROVED** [review]

**Comment by divipillai** [comment]
This PR can be merged.

**Review by anangl — APPROVED** [review]

**Pull request merged**